ceci/stage.py

```diff
@@ -1284,7 +1298,15 @@ def generate_command(
         module = cls.get_module()
         module = module.split(".")[0]
 
-        flags = [cls.name]
+        if sys.modules[module].__file__:
```

Comment:
I don't quite understand this bit - can you explain in a comment?  When is the `__file__` attribute of a module non-empty?

Reply:
Ok, I clarified the comment